.github/workflows/manual-build.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
What does this action do? Could you add some docs?

Reply:
It allows you to build an image on demand by manually running a workflow from the github actions tab. The docs will be added at some point to https://github.com/kbase/.github/tree/main/guide